R/stats.R

```diff
@@ -0,0 +1,91 @@
+#' Summary statistic functions
+#'
+#' `{messydates}` provides extensions to a variety of functions
+#' with R's `{stats}` package to simplify the creation of
+#' summary reports of data with `{skimr}`.
+#' All functions take a `mdate` vector as an input and return a single
+#' statistic.
+#' Note that individual dates are first expanded and resolved
+#' before a vector level statistic is computed
+#' (e.g. `maxmax()` corresponds to the maximum date of the maximum
+#' of the vector resulting from an expanded `{mdate}`)
+#' @seealso [get_skimmers()] for customizing the functions called by [skim()].
```

Comment:
Do we really need to document `get_skimmers.mdate()`? You can export without documenting... Also, do we really need to point to `skimr::get_skimmers()` at all?

Reply:
I removed the point to `skimr::get_skimmers()` from documentation. However, we do need to document `get_skimmers.mdate()` to properly return report with correct date class.

Reply:
You can just export it without documenting it